### PR TITLE
Windows instructions updated in harmonic branch

### LINKED ISSUE #483: Windows instructions should be updated
As of https://github.com/gazebosim/gz-sim/pull/2382, Gazebo actually runs with Ogre2 on windows, but our [documentation](https://gazebosim.org/docs/latest/install_windows_src/) states that only Ogre (1) is supported. The ["GUI workaround"](https://gazebosim.org/docs/latest/install_windows_src/#gazebo-gui-workaround
) section also mentions that `gz sim -g` does not work.

The [binary](https://gazebosim.org/docs/latest/install_windows/) install instructions also say:
> Additionally, command line tools, the DART physics engine, and some tests are not currently supported in Windows.

which is no longer accurate.

### PR DESCRIPTION
**Pull request by akky20**
# 🦟 Bug fix

Fixes #483 

## Summary
Changes doc text according to the mentioned parts in issue

## Checklist
- [ ] Signed all commits for DCO
- [ ] Added tests
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: harmonic/install_windows_src.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ WARNING: Current Windows support is experimental.
 
 # Source Installation on Windows 10 or 11
 
-OGRE2 rendering capabilities are not currently supported in Windows, and Gazebo GUI
+OGRE2 rendering capabilities are also supported in Windows, and Gazebo GUI
```
Comment (azeey) [inline harmonic/install_windows_src.md:7]: Do we need the "also"? I think we can just say "are supported".

File: harmonic/install_windows_src.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ WARNING: Current Windows support is experimental.
...
 works in a limited fashion. These functionalities correspond to the currently
 building packages `gz-rendering` and `gz-sim`, respectively. The packages will build,
```
Comment (azeey) [inline harmonic/install_windows_src.md:9]: Probably update the line that says you'll expect failures.

**Review by azeey — COMMENTED** [review]
Thanks for the contribution! I left some small comments.
There's also a note on line 145 that says Gazebo GUI is not yet working which needs to be updated.

You can also remove the line that says "Manually comment these lines"

**Comment by akky20** [comment]
> Thanks for the contribution! I left some small comments. There's also a note on line 145 that says Gazebo GUI is not yet working which needs to be updated.
> 
> You can also remove the line that says "Manually comment these lines"

Done with the changes

**Comment by akky20** [comment]
@azeey Please look at the changes

File: harmonic/install_windows_src.md [hunk 3]
```diff
@@ -143,7 +143,11 @@ This is the end of the source install instructions; head back to the [Getting st
 page to start using Gazebo!
 
 > **NOTE**
-> As Gazebo GUI is not yet working, running `gz sim` will not work. You can run only the server with
+> To launch Gazebo GUI , run 
+>```bat
+>gz sim
```
Comment (traversaro) [inline harmonic/install_windows_src.md:148]: I do not think this works (see https://github.com/gazebosim/gz-sim/issues/2393). What is currently running is `gz sim -s` and `gz sim -g` separately, a bit like on macOS, I guess we can take the macOS documentation as example?

Comment (akky20) [reply harmonic/install_windows_src.md]: Yes My mistake , corrected it

**Review by azeey — APPROVED** [review]

**Pull request merged**